service/src/main/kotlin/fi/espoo/evaka/shared/dev/DataInitializers.kt

```diff
@@ -439,16 +431,12 @@ fun Database.Transaction.insertTestPlacement(
             VALUES (:id, :childId, :unitId, :type::placement_type, :startDate, :endDate)
             """
     )
-        .bindMap(
-            mapOf(
-                "id" to id,
-                "childId" to childId,
-                "unitId" to unitId,
-                "type" to type.toString(),
-                "startDate" to startDate,
-                "endDate" to endDate
-            )
-        )
+        .bind("id", id)
+        .bind("childId", childId)
+        .bind("unitId", unitId)
+        .bind("type", type.toString())
```

Comment:
Is there a reason you left this `.toString()` in? Couldn't `PlacementType` implement `DatabaseEnum`?

Reply:
I just missed this :sweat_smile: